generic/shared/src/main/scala/monocle/generic/Generic.scala

```diff
@@ -0,0 +1,13 @@
+package monocle.generic
+
+import monocle.Iso
+import shapeless.Generic
+
+object generic extends GenericIso
+
+trait GenericIso {
```

Comment:
I wanted to name this trait `GenericInstances` to match the other traits, but unfortunately that name was already taken :/

Reply:
maybe we could call it `GenericOptics` in case we implement something else than `Iso` later but it doesn't really matter

Reply:
done, thanks!